php-classes/Slate/Courses/SectionTermData.php

```diff
@@ -1,48 +1,52 @@
 <?php
 
-namespace Slate\Progress\Narratives;
+namespace Slate\Courses;
 
-class SectionNotes extends \VersionedRecord
+class SectionTermData extends \VersionedRecord
 {
     // VersionedRecord configuration
-    public static $historyTable = 'history_narrative_section_notes';
+    public static $historyTable = 'history_section_term_data';
 
     // ActiveRecord configuration
-    public static $tableName = 'narrative_section_notes';
-    public static $singularNoun = 'narrative section note';
-    public static $pluralNoun = 'narrative section notes';
-    public static $collectionRoute = '/progress/narratives/section-notes';
+    public static $tableName = 'section_term_data';
+    public static $singularNoun = 'section term datum';
+    public static $pluralNoun = 'section term notes';
+    public static $collectionRoute = '/section-data';
     public static $updateOnDuplicateKey = true;
     public static $trackModified = true;
 
     public static $fields = [
-        'CourseSectionID' => [
+        'SectionID' => [
             'type' => 'integer',
             'unsigned' => true
         ],
         'TermID' => [
             'type' => 'integer',
             'unsigned' => true
         ],
-        'Notes' => [
+        'TermReportNotes' => [
             'type' => 'clob',
-            'notnull' => false
+            'default' => null
+        ],
+        'InterimReportNotes' => [
+            'type' => 'clob',
+            'default' => null
         ]
     ];
 
 
     public static $indexes = [
-        'NarrativeSectionNote' => [
-            'fields' => ['TermID', 'CourseSectionID'],
+        'SectionTermData' => [
+            'fields' => ['TermID', 'SectionID'],
             'unique' => true
         ]
     ];
 
     public static $relationships = [
         'Section' => [
             'type' => 'one-one',
-            'class' => \Slate\Courses\Section::class,
-            'local' => 'CourseSectionID'
+            'class' => Section::class,
+            'local' => 'SectionID'
```

Comment:
@nbey setting `local` here shouldn't be necessary as this matches relationshipName+'ID'